install-components.md

```diff
@@ -1720,155 +1754,11 @@ To install Supply Chain Security Tools - Sign:
     Added installed package 'image-policy-webhook' in namespace 'tap-install'
     ```
 
-   After you run the code above, the webhook is running.
+   After you run the commands above the webhook will be running.
```

Comment:
Do you feel like `with the required configuration` may imply configuration is done and no further action is required?

We have a note right after this paragraph that states our product requires more configuration and I'm afraid that if we say "required configuration" here it might give the impression that no further action from the platform operator is required.